WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -76,11 +76,6 @@ class NewGutenbergViewController: UIViewController, PostEditor, PublishingEditor
         return !changes.isEmpty
     }
 
-    // TODO: this has to be incorrect and/or lagging behind
-    var editorHasContent: Bool {
-        !editorViewController.state.isEmpty
-    }
```

Comment:
As noted in the comment, this was often incorrect. This was removed to rely upon the existing `editorHasContent` [implementation](https://github.com/wordpress-mobile/WordPress-iOS/blob/d3e49882603219af9756c8d41d381bf3cc01abe4/WordPress/Classes/ViewRelated/Post/PostEditor.swift#L77-L79), which appears to be more accurate.